composer.json

```diff
@@ -2,17 +2,18 @@
     "name": "mittwald/vault-php",
     "type": "library",
     "license": "MIT",
-    "version": "0.0.1",
+    "version": "0.0.2",
```

Comment:
Typically, we shouldn't need a `"version"` in the composer.json at all, since Composer is smart enough to just work with Git tags, anyway. Maybe remove it altogether?

Reply:
True thing, i removed the `version`.